### DISCUSSION TITLE
AB Scripting - Adding media items to media item records task

### DISCUSSION THREAD

**Discussion by longj08** [sadieblue/agilityblue.workspace-scripts#15] [Q&A] [answered]
I would like to add media items check in to a media item records task.  The media item should be located at Form->Sections->Fields->ReferenceObject->Values.  Is there a sample script of how I can achieve this?  For example, I would like to add "#70 - Hard Drive" to the below list.

Also is it possible to rename the media item name after it is added? For example, I would like to rename the below media item of "#34 - Hard Drive" to "01 - Hard Drive."

![image](https://github.com/user-attachments/assets/7cd21bf8-9ee3-4236-a316-c0c794aac944)

Any help is appreciated. Thank you!

**Comment by bmward80** [comment, marked as answer]
Hi, @longj08,

We don't have any specific examples of that scenario, but you may find useful information about working with reference fields in our help documentation here: [https://help.agilityblue.com/docs/scripting-tutorials-custom-fields-deep-dive#data-type-reference](https://help.agilityblue.com/docs/scripting-tutorials-custom-fields-deep-dive#data-type-reference). The examples are geared towards objects, but the concepts and structure are the same for the fields list on task forms. That link should also help answer your question about whether you can rename the display value of reference fields or not.

Hope this helps!
Brandon
